common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java

```diff
@@ -69,7 +68,7 @@ public void testInterception() throws Exception {
       decoder.channelRead(ctx, len);
       decoder.channelRead(ctx, dataBuf);
       verify(interceptor, times(interceptedReads)).handle(any(ByteBuf.class));
-      verify(ctx).fireChannelRead(any(ByteBuffer.class));
+      verify(ctx).fireChannelRead(any(ByteBuf.class));
```

Comment:
Hi, @vanzin .
Could you review this? The other place seems to use `ByteBuf` consistently.
```
$ git grep 'fireChannelRead(any(ByteBuf.class)'
common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java:      verify(ctx, never()).fireChannelRead(any(ByteBuf.class));
common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java:      verify(ctx).fireChannelRead(any(ByteBuf.class));
common/network-common/src/test/java/org/apache/spark/network/util/TransportFrameDecoderSuite.java:      verify(ctx, times(frameCount)).fireChannelRead(any(ByteBuf.class));
```

Reply:
I was trying to figure out why this even compiled (the class doesn't really make a difference here otherwise, since it's an `any` matcher). The API is actually `fireChannelRead(Object msg)`, which explains it.

Anyway, the change is fine.

Reply:
Thank you!